lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -10,6 +10,9 @@ class AnsibleRunner < ::Proxy::Dynflow::Runner::Parent
       include ::Proxy::Dynflow::Runner::ProcessManagerCommand
       attr_reader :execution_timeout_interval
 
+      # To make this overridable in development
+      ENVIRONMENT_WRAPPER = ENV['SMART_PROXY_ANSIBLE_ENVIRONMENT_WRAPPER'] || '/usr/libexec/ansible-runner-environment'
```

Comment:
Should this be placed in `/usr/libexec/foreman-proxy` instead? We already have `/usr/libexec/foreman-proxy/puppet_sign.rb` there (from the foreman-proxy package).
```suggestion
      ENVIRONMENT_WRAPPER = ENV['SMART_PROXY_ANSIBLE_ENVIRONMENT_WRAPPER'] || '/usr/libexec/foreman-proxy/ansible-runner-environment'
```